modules/javafx.graphics/src/main/java/javafx/print/PrinterJob.java

```diff
@@ -209,28 +209,8 @@ public String getName() {
     }
 
     /**
-     * Property representing the
-     * <code>Printer</code> for this job.
-     * @return the <code>Printer</code> for this job
-     */
-    public final ObjectProperty<Printer> printerProperty() {
-        /* The PrinterJob constructor always creates this property,
-         * so it can be returned directly.
-         */
-        return printer;
-    }
-
-    /**
-     * Gets the printer currently associated with this job.
-     * @return printer for the job.
-     */
-    public synchronized Printer getPrinter() {
-        return printerProperty().get();
-    }
-
-    /**
-     * Change the printer for this job.
-     * If the new printer does not support the current job settings,
+     * Property representing the {@code Printer} for this job.
```

Comment:
I guess we normally used <code></code> throughout . Probably better to be consistent.

Reply:
Using `{@code xxx}` is preferred to `<code>xxx</code>` so for new fixes we might as well use the preferred pattern.